controllers/ControllerAdmin.php

```diff
@@ -3,201 +3,212 @@
-			                                           'server' => $this->_server,
-			                                           'user'=> $key['pseudo'],
-			                                           'value'=> $key['email'],
-			                                           'avatar'=> $key['avatar']]);
+			echo $this->getTwig()->render('profile.twig', ['logged'=> TRUE,
+			                                               'server' => $this->getServer(),
+			                                               'user'=> $key['pseudo'],
+			                                               'value'=> $key['email'],
+			                                               'admin' => $key['admin'],
+			                                               'avatar'=> $key['avatar']]);
 		}
 
 		else
 			{
-			$validChange = $this->_userManager->updateUserMail($params['identifiant'], $key);
+			$validChange = $userManager->updateUserMail($params['identifiant'], $key);
+
 			if ($validChange){
 				(new Session)->setValueKey('user', 'email', $params['identifiant']);
 				$key['email'] = $params['identifiant'];
-				echo $this->_twig->render('profile.twig', ['confirm' => TRUE,
-				                                           'server' => $this->_server,
-				                                           'logged' => TRUE,
-				                                           'user' => $key['pseudo'],
-				                                           'value' => $key['email'],
-				                                           'avatar'=> $key['avatar']]);
+
+				echo $this->getTwig()->render('profile.twig', ['confirm' => TRUE,
+				                                               'server' => $this->getServer(),
+				                                               'logged' => TRUE,
+				                                               'user' => $key['pseudo'],
+				                                               'value' => $key['email'],
+				                                               'admin' => $key['admin'],
+				                                               'avatar'=> $key['avatar']]);
 
 			}
 		}
-
-
-
 	}
 
+	/**
+	 * Vérifie que les mots de passe correspondent et le met à jour dans la BDD.
+	 *
+	 * Page Profil
+	 */
+
 	public function checkPassProfile(){
-		$params = $this->_params;
+		$params = $this->getPost();
 		$key = (new Session)->getKey('user');
 
 		$this->checkForm($params);
 
 		if($params['password'] != $params['confirme']){
-			$valid = true;
-			echo $this->_twig->render('profile.twig', ['valid' => $valid,
-			                                           'server' => $this->_server,
-			                                           'logged'=> TRUE,
-			                                           'user'=> $key['pseudo'],
-			                                           'value'=> $key['email'],
-			                                           'avatar'=> $key['avatar']]);
+
+			echo $this->getTwig()->render('profile.twig', ['valid' => TRUE,
```

Comment:
Codacy found an issue: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$this'.](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=6061935)